userspace/libsinsp/test/container_info.ut.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright (C) 2021 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#include "container_info.h"
+#include <gtest/gtest.h>
+#include <tuple>
+#include <vector>
+
+class sinsp_container_lookup_test : public ::testing::TestWithParam<std::tuple<short, short, std::vector<short>>>
```

Comment:
Classes and structs are exactly the same in C++, apart from the visibility of their memebers (public by default for classes, private by default for structs). In this case where there is nothing to define private or public you can just use struct and avoid the `public` keyword, thus making it more readable.
```c++
struct sinsp_container_lookup_test : 
    ::testing::TestWithParam<std::tuple<short, short, std::vector<short>>>
{
};
```